mne/annotations.py

```diff
@@ -474,7 +474,7 @@ def _annotations_starts_stops(raw, kinds, name='unknown', invert=False):
         for kind in kinds:
             _validate_type(kind, 'str', "All entries")
 
-    if len(raw.annotations) == 0:
+    if not raw.annotations:
```

Comment:
we had discussed this is the past. It's more explicit with len and it also makes sure that raw.annotations is never None

Reply:
I thought of using its truth value, because with `if not raw.annotations` it's crystal clear what you mean. I'd be in favor of using it like that, but if you've already discussed and decided against it then OK. Regarding guarding against `None`, we could make sure that annotations are always initialized with an empty `Annotations` object and never with `None`.

Reply:
with `if len(raw.annotations) == 0:` you shoot 2 bids with one stone: you make sure htere is no annotations and it's not None

Reply:
Sure, but it's also not very Pythonic. I agree that the `None` check is nice, but if we made sure that `raw.annotations` is never `None` this would not be necessary.